.travis.yml

```diff
@@ -67,6 +67,7 @@ install:
 
 script:
   - if [[ $TEST_COVERAGE == 'true' ]]; then composer test-coverage ; else composer test ; fi
+  - if [[ $TEST_COVERAGE == 'true' ]]; then composer static-analysis ; fi
```

Comment:
Is checking during code coverage expected, or did you mean to use a different variable?

Reply:
I copied it from laminas-cli, but I think it would make more sense to create separate and run all these check with locked deps.